fetch.bs

```diff
@@ -7533,7 +7533,7 @@ therefore not shareable, a WebSocket connection is very close to identical to an
 <h3 id=websocket-opening-handshake>Opening handshake</h3>
 
 <p>To <dfn id=concept-websocket-establish>establish a WebSocket connection</dfn>, given a
-<var>url</var>, <var>protocols</var>, and <var>client</var>, run these steps:
+<var>url</var>, <var>protocols</var>, <var>credentials mode</var>, and <var>client</var>, run these steps:
```

Comment:
```suggestion
<var>url</var>, <var>protocols</var>, <var>credentialsMode</var>, and <var>client</var>, run these steps:
```
and the same below. That makes it slightly clearer it's a single variable and not two.